app/javascript/packs/simulator.js

```diff
@@ -0,0 +1,7 @@
+import 'jquery-ujs';
+
+import '../../../public/src/app';
```

Comment:
Webpacker files shouldn't on public folder, move them to app/javascripts

Reply:
@shubhankarsharma00

Reply:
@tachyons actually since finally the plan is to move the codebase in public to a standalone github repo and use it as submodule which I think would be best imported in public.

Reply:
Because the standalone repo will include CSS and images to make a standalone bundle.

Reply:
public is a special folder in rails where everything inside will be exposed as it is, so having the source and config in public can cause accidentally exposing credentials in public